examples/cross-contract-call-tracing/lib.rs

```diff
@@ -40,8 +40,8 @@ mod contract {
         #[ink(message)]
         pub fn middle_call(&self, next_callee: AccountId, arg: u32) -> u32 {
             build_call::<DefaultEnvironment>()
-                .call(next_callee)
-                .gas_limit(0)
+                .call_v1(next_callee)
+                .gas_limit(1_000_000)
```

Comment:
That's probably not enough gas, try at least `500_000_000`. Or just leave it out completely it will inherit the gas limit of the top level call.